src/components/Calendar/Calendar.tsx

```diff
@@ -49,15 +52,71 @@ const getViewsFromViewsProp = (views: View[]) => {
 }
 
 const Calendar = (props: Props) => {
-  const { view, views, events, disabled, onDateClick, onDateRangeSelected, onEventClick } = props
+  const {
+    view,
+    views,
+    events,
+    disabled,
+    onDateClick,
+    onDateRangeSelected,
+    onEventClick,
+    onPrevClick,
+    onNextClick,
+    onTodayClick,
+  } = props
   const fullCalendarRef = React.createRef<FullCalendar>()
+
+  const onNavClick = (to: 'prev' | 'next' | 'today') => {
+    const calendar = fullCalendarRef?.current?.getApi()
+    if (!calendar) {
+      return
+    }
+
+    const toCalls = {
+      prev: {
+        nav: () => calendar.prev(),
+        callback: onPrevClick,
+      },
+      next: {
+        nav: () => calendar.next(),
+        callback: onNextClick,
+      },
+      today: {
+        nav: () => calendar.today(),
+        callback: onTodayClick,
+      },
+    }
+
+    const { nav, callback } = toCalls[to]
+    nav()
+    if (callback !== undefined) {
+      callback()
+    }
+  }
+
   return (
     <FullCalendar
       events={events}
       ref={fullCalendarRef}
       selectable={!disabled}
+      customButtons={{
+        customPrev: {
+          text: 'Prev',
+          icon: 'chevron-left',
+          click: () => onNavClick('prev'),
+        },
+        customNext: {
+          text: 'Next',
+          icon: 'chevron-right',
+          click: () => onNavClick('next'),
+        },
+        customToday: {
+          text: 'today',
```

Comment:
I think we should have capital `Today` instead of `today`

Reply:
The right side of the header has its buttons, day/week/month, in lower case. So I think it'd be more consistent with today being lower case too?

Reply:
ah, you're right. ignore this